client/my-sites/site-settings/theme-setup/theme-setup-card.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import ActionPanel from 'my-sites/site-settings/action-panel';
+import ActionPanelTitle from 'my-sites/site-settings/action-panel/title';
+import ActionPanelBody from 'my-sites/site-settings/action-panel/body';
+import ActionPanelFooter from 'my-sites/site-settings/action-panel/footer';
+import ActionPanelFigure from 'my-sites/site-settings/action-panel/figure';
+import Notice from 'components/notice';
+import Button from 'components/button';
+import ActiveThemeScreenshot from './active-theme-screenshot';
+
+const ThemeSetupCard = ( { theme, translate } ) => {
+	return (
+		<ActionPanel>
+			<ActionPanelBody>
+				<ActionPanelTitle>{ translate( 'Theme Setup' ) }</ActionPanelTitle>
+				<Notice status={ 'is-warning' } showDismiss={ false }>
+					{ translate( 'This action cannot be undone.' ) }
```

Comment:
Alternate string suggestion: 
* `translate( 'This cannot be undone' )`&mdash; translations: **3**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/settings-theme-setup)